packages/polymorphic/README.md

```diff
@@ -0,0 +1,148 @@
+# Polymorphic
+
+> ### **pol·y·morph** _(noun)_
```

Comment:
General question - should we name this package `Polymorph`? and `Polymorphic` and other resources can be utilities around Polymorph? 

Feels nicer since it'd be a noun and the relationship ^ between Polymorph and Polymorphic feels more intuitive to me with that change.

Reply:
That's a fair point. But I think the package name `polymorphic` feels more natural, especially since the `Polymorphic` function will likely be more commonly used than the standalone `Polymorph` component